h2o-r/h2o-package/R/isolationforest.R

```diff
@@ -48,7 +48,7 @@
 #' cars <- h2o.importFile(f)
 #' 
 #' # Set the predictors
-#' predictors <- c("displacement","power","weight","acceleration","year")
```

Comment:
This is the wrong file to update. The changes to this and other algos have to be made in:
/h2o-3/h2o-bindings/bin/custom/R